src/recognition/ReadIDX.java

```diff
@@ -0,0 +1,313 @@
+/**
+ *
+ * @file
+ * @brief Here is wrapper class (reader) for IDX files which contains
+ * images with numbers examples.
+ *
+ *
+ * @authors Vladislav Shikhanov
+ *
+ *		  http://move-llc.ru
+ *****************************************************************************/
+
+package recognition;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+
+
+
```

Comment:
It is good that you write such well-documented wrapper. But try to use a smaller number of empty lines.